### PR TITLE
`_revertIfActiveAuctions` helper should revert with `ActiveAuction` error

### PR DESCRIPTION
**Pull request by grandizzy**
## Description

accommodating comment https://github.com/ajna-finance/contracts/pull/1024#pullrequestreview-1777683604

## Impact

slightly increase of contract size as new error used

## Tasks

- [ ] Changes to protocol contracts are covered by unit tests executed by CI.
- [ ] Protocol contract size limits have not been exceeded.
- [ ] Gas consumption for impacted transactions have been compared with the target branch, and nontrivial changes cited in the _Impact_ section above.
- [ ] Scope labels have been assigned as appropriate.
- [ ] Invariant tests have been manually executed as appropriate for the nature of the change.

### CODE REVIEW

**Review by EdNoepel — APPROVED** [review]
LGTM.

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Review by prat-gpt — APPROVED** [review]
LGTM

File: src/base/Pool.sol [hunk 1]
```diff
@@ -396,7 +396,7 @@ abstract contract Pool is Clone, ReentrancyGuard, Multicall, IPool {
      *  @dev    update `reserveAuction.latestBurnEventEpoch` and burn event `timestamp` state
      *  @dev    === Reverts on ===
      *  @dev    2 weeks not passed `ReserveAuctionTooSoon()`
-     *  @dev    unsettled liquidation `AuctionNotCleared()`
+     *  @dev    unsettled liquidation `ActiveAuction()`
```
Comment (ith-harvey) [inline src/base/Pool.sol]: shouldn't this comment match the error and instead say `AuctionActive()`?

**Review by ith-harvey — COMMENTED** [review]
suggested comment change

File: src/base/Pool.sol (hunk 1 above)
Comment (grandizzy) [reply src/base/Pool.sol]: fixed in https://github.com/ajna-finance/contracts/pull/1031/commits/88fb05e3ccc27621cc9aafbdd43c41e8ba1aae5e

**Review by dmitriia — COMMENTED** [review]
Looks ok

**Pull request merged**